ui/app/registration/views/header.html

```diff
@@ -1,6 +1,6 @@
 <header class="reg-header" ng-controller="NavigationController">
         <ul class="top-nav fl">
-            <a class="back-btn" accesskey="h" href="../home/index.html"><i class="fa fa-home"></i></a>
+            <a class="back-btn" accesskey="h" href="../home/index.html" ng-click="onHomeNavigate()" id="bahmniHome"><i class="fa fa-home"></i></a>
```

Comment:
Why is `id="bahmniHome"` needed in the end?

Reply:
Actually its no longer needed now.I'll remove it